.gitignore

```diff
@@ -339,4 +339,6 @@ ASALocalRun/
 .localhistory/
 
 # BeatPulse healthcheck temp database
-healthchecksdb
\ No newline at end of file
+healthchecksdb
+MFaaP.MFWSClient-with-sandbox.sln
```

Comment:
Not sure what this .sln and Sandbox items are.  They are in the ignore but not in the source; are these specific to your scenario?  If so then please remove these from the solution and update the gitignore for clarity.